internal/avs/external_eval_assistant.go

```diff
@@ -68,7 +68,7 @@ func (eea *ExternalEvalAssistant) ProvideTags(operation internal.Operation) []*T
 			TagClassId: eea.avsConfig.LandscapeTagClassId,
 		},
 		{
-			Content:    string(operation.ProvisioningParameters.PlatformProvider),
+			Content:    providerCodeByPlan(operation.ProvisioningParameters.PlanID),
```

Comment:
The `providerCodeByPlan` returns "AZR" for azure plan, however our provider tag requires "Azure". 
You might need to introduce a new mapping function in internal/avs package to map plan ID to provider tag.

Reply:
ah youre right. i was not thinking about this. aws and gcp it works but not for azure.

Reply:
ive made my changes.